src/Types.php

```diff
@@ -26,6 +26,46 @@ class Types
     const TYPE_STRING = 'string';
 
     /**
+     * @var array
+     */
+    public static $phpToJson = [
+        'array' => self::TYPE_ARRAY,
+        'boolean' => self::TYPE_BOOLEAN,
+        'double' => self::TYPE_NUMBER,
+        'integer' => self::TYPE_INTEGER,
```

Comment:
Those arrays are there mostly to replace the slow switch statements (with linear complexity) with hash lookups (with constant complexity).